src/general/GameWorld.cs

```diff
@@ -92,6 +93,13 @@ public GameWorld(WorldGenerationSettings settings, Species? startingSpecies = nu
         GenerationHistory.Add(0, new GenerationRecord(
             0,
             new Dictionary<uint, SpeciesRecordLite> { { PlayerSpecies.ID, initialSpeciesRecord } }));
+
+        if (WorldSettings.DayNightCycleEnabled)
+        {
+            // Make sure average light levels are computed already
+            foreach (var patch in Map.Patches)
+                patch.Value.UpdateAverageSunlight(LightCycle);
```

Comment:
I think this should be made into a general method in the patch map just for that extra bit of clarity and guarantee against accidentally modifying just one copy of this code (I saw this same kind of looking pattern in the editor).

Reply:
I put this in `GameWorld` to reduce unnecessary passing of `DayNightCycle` in methods.